pkg/component/shared/istio_test.go

```diff
@@ -322,7 +326,8 @@ var _ = Describe("Istio", func() {
 					labels,
 					&externalTrafficPolicy,
 					serviceExternalIP,
-					zone)).To(Succeed())
+					zone,
+					false)).To(Succeed())
```

Comment:
How about adding a test for `dualStack=true`?

Reply:
Test added.